static/app/views/settings/account/accountClose.tsx

```diff
@@ -147,16 +149,17 @@ function AccountClose() {
       </Alert>
 
       <Panel>
-        <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+        <PanelHeader>{t('Delete the following organizations')}</PanelHeader>
         <PanelBody>
-          <PanelAlert type="info">
+          <PanelAlert type="warning">
+            <strong>{t('ORGANIZATIONS WITH CHECKED BOXES WILL BE DELETED!')}</strong>
+            <br />
             {t(
               'Ownership will remain with other organization owners if an organization is not deleted.'
             )}
             <br />
-            {tct(
-              "Boxes which can't be unchecked mean that you are the only organization owner and the organization [strong:will be deleted].",
-              {strong: <strong />}
+            {t(
+              "Boxes which can't be unchecked mean that you are the only organization owner and the organization will be deleted."
```

Comment:
```suggestion
            {t(
              "Boxes which can't be unchecked mean that you are the only organization owner and the organization [strong:will be deleted]."
```
nit: i think we can still keep this

Reply:
honestly could go either way, but I do think the new header says the same thing and users should prob read that over anything else in the warning box (don't really want to detract from the header).